downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -55,16 +57,18 @@ public static void processCellularComponents(GKInstance reactionInst) throws Exc
      * @param taxonIdentifier -- String, CrossReference ID of protein's species.
      * @throws Exception -- MySQLAdaptor exception.
      */
-    private static void generateGOCellularCompartmentLine(GKInstance proteinInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
+    private static String generateGOCellularCompartmentLine(GKInstance proteinInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
         String reactomeIdentifier = REACTOME_IDENTIFIER_PREFIX  + GOAGeneratorUtilities.getStableIdentifierIdentifier(reactionInst);
         GKInstance referenceEntityInst = (GKInstance) proteinInst.getAttributeValue(ReactomeJavaConstants.referenceEntity);
         String goCellularCompartmentAccession = getCellularCompartmentGOAccession(proteinInst);
+        String goaLine = null;
         if (!goCellularCompartmentAccession.isEmpty()) {
-            String goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, CELLULAR_COMPONENT_LETTER, goCellularCompartmentAccession, reactomeIdentifier, TRACEABLE_AUTHOR_STATEMENT_CODE, taxonIdentifier);
+            goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, CELLULAR_COMPONENT_LETTER, goCellularCompartmentAccession, reactomeIdentifier, TRACEABLE_AUTHOR_STATEMENT_CODE, taxonIdentifier);
```

Comment:
Similar situation to the comment above with the last iteration's value in the loop being the only one returned.

Reply:
In this case there isn't any iteration taking place

Reply:
Ah, you are correct!  I must've misread it!